cmd/sentinel/cmd/sentinel.go

```diff
@@ -379,6 +379,8 @@ func (s *Sentinel) setDBSpecFromClusterSpec(cd *cluster.ClusterData) {
 		db.Spec.RequestTimeout = *clusterSpec.RequestTimeout
 		db.Spec.MaxStandbys = *clusterSpec.MaxStandbys
 		db.Spec.UsePgrewind = *clusterSpec.UsePgrewind
+		db.Spec.PgrewindInterval = *clusterSpec.PgrewindInterval
```

Comment:
The case of these is a little sad but just following the convention used elsewhere.